### ISSUE TITLE
Projecting 3D Bbox onto image

### ISSUE THREAD

**Issue by jayjo217** [naurril/SUSTechPOINTS#96]
Hi.
I have a question in the process of projecting the 3D Bbox created on the LiDAR screen onto the image.

When the corner points of the 3D Bbox is converted from the LiDAR coordinate system to the camera coordinate system, I wonder what processing is done in the process of projecting the negative value based on the Z axis to the image.

In the code, it seems that the point is ignored, but in the window shown at the bottom left when editing the 3D Bbox, it was shown to be displayed the same as the point having a positive value based on the Z axis on the camera coordinates.

Thank you in advance.
Sincerely,

**Comment by kanelian63** [comment]
I have a same question

**Comment by naurril** [comment]
Yes, if the box is behind the camera (z<0) it's ignored. The difference is in bottom left subview, the box can be drawn outside of the image (z > 0，x or y outside image range )
